ios/FluentUI/Core/SwiftUI+ViewPresentation.swift

```diff
@@ -21,6 +21,7 @@ struct PresentationModifier<PresentedView: View>: ViewModifier {
             .allowsHitTesting(isPresented && isBlocking)
 
             presentedView
+                .clipped()
```

Comment:
Shouldn't this modifier be applied after the frame is defined? (after line 28)

Reply:
Hm, I thought it was the presentedView (the notification in this case) that we wanted to be clipped if it was outside of the ZStack versus the whole ZStack being clipped (though after testing both ways, I don't see any differences)

Reply:
I think it's a bit confusing and I ended up going to the [clipped(antialiased:) documentation](https://developer.apple.com/documentation/swiftui/path/clipped(antialiased:)) again:

My understanding is that it will make the ZStack to clip everything out of its frame:
`A view that clips this view to its bounding frame.`

The counterpart API in UIKit ([clipsToBounds](https://developer.apple.com/documentation/uikit/uiview/1622415-clipstobounds)) has a clearer explanation:
`Setting this value to true causes subviews to be clipped to the bounds of the receiver. If set to false, subviews whose frames extend beyond the visible bounds of the receiver are not clipped. The default value is false.`

I did test it myself and observed the same as you did (no difference). I believe that is probably because SwiftUI understands that the bounding frame of the **presentedView** is the ZStack frame.